tensorboardX/writer.py

```diff
@@ -351,12 +352,12 @@ def add_hparams(self, hparam_dict=None, metric_dict=None):
             raise TypeError('hparam_dict and metric_dict should be dictionary.')
         exp, ssi, sei = hparams(hparam_dict, metric_dict)
 
-        with SummaryWriter(logdir=os.path.join(self.file_writer.get_logdir(), str(time.time()))) as w_hp:
```

Comment:
str(time.time()) will be decided at compile time. So If you are depending on the default parameter, you will end up with only one result. Also, there are syntax errors in the code.

Reply:
Ok, edited with regards to the param init. 
Which syntax errors are you referring to?

Reply:
@lanpa the time is actually not decided at compile time in python, so you end up with multiple different sub-directories named after the timestamp - just tested it. I think this PR is really a valuable addition to tensorboardX once the syntax error is fixed.